### ISSUE TITLE
[node] Call to getStateChannel failed when searching for multisig address

### ISSUE THREAD

**Issue by joelalejandro** [counterfactual/monorepo#2275]
### Error

After #2272 was closed, E2E tests began to fail on `master` due to the following:

```
[7733:7733:0826/170237.438572:INFO:CONSOLE(1)] "Error: Call to getStateChannel failed when searching for multisig address: 0x84c4D570259d7EB4E41b30f6794bAA46a566CBa9. This probably means that the StateChannel does not exist yet.
    at Store.getStateChannel (chrome-extension://bhiaejihpgjcelfaimbfkacnjenlklnb/vendor/counterfactual/node/node.iife.js:3871:23)
    at async DepositController.beforeExecution (chrome-extension://bhiaejihpgjcelfaimbfkacnjenlklnb/vendor/counterfactual/node/node.iife.js:4975:29)
    at async DepositController.executeMethod (chrome-extension://bhiaejihpgjcelfaimbfkacnjenlklnb/vendor/counterfactual/node/node.iife.js:3507:13)
    at async RpcRouter.dispatch (chrome-extension://bhiaejihpgjcelfaimbfkacnjenlklnb/vendor/counterfactual/node/node.iife.js:5194:29)
    at async t.exports.metamaskRequestDepositRPC (chrome-extension://bhiaejihpgjcelfaimbfkacnjenlklnb/background.js:1:152694)
    at async chrome-extension://bhiaejihpgjcelfaimbfkacnjenlklnb/background.js:1:155882", source: chrome-extension://bhiaejihpgjcelfaimbfkacnjenlklnb/background.js (1)
[7733:7733:0826/170237.439551:INFO:CONSOLE(1)] "Error: Error: Call to getStateChannel failed when searching for multisig address: 0x84c4D570259d7EB4E41b30f6794bAA46a566CBa9. This probably means that the StateChannel does not exist yet.", source: chrome-extension://bhiaejihpgjcelfaimbfkacnjenlklnb/background.js (1)
[7733:7733:0826/170237.450859:INFO:CONSOLE(69423)] "wallet error", source: http://localhost:3334/static/js/0.chunk.js (69423)
```

### Steps to reproduce
1. Sync everything with master.
2. Edit `packages/greenboard/test-sequencer.js:8` to have this line:
    ```ts
    const testSequence = ["tests/onboarding.spec.ts"];
    ```
3. Run the following in a console, from the root's monorepo:
    ```shell
    yarn test:e2e:setup
    NODE_EXTENDED_PRIVATE_KEY=xprv... yarn test:e2e
    ```
4. Observe the automation run. It'll trigger an "Ups! Something broke" message on the deposit form.

#### Account creation
```
@counterfactual/simple-hub-server: [timestamp:2019-08-26T20:02:27.490Z] [level:info] [message:User has been created] [username:teapot] [userId:undefined] [endpoint:createAccount]
@counterfactual/simple-hub-server: [timestamp:2019-08-26T20:02:27.495Z] [level:info] [message:User token has been generated] [endpoint:createAccount]
@counterfactual/simple-hub-server: [timestamp:2019-08-26T20:02:27.501Z] [level:info] [message:IncomingRequest] [url:/api/users] [queryParams:undefined] [body:{"data":{"attributes":{"username":"teapot","email":"i.am.a@tea.pot","ethAddress":"0x212c90fdf90bbd5e9b352b9d2b086f2666cfeed6","nodeAddress":"xpub6Dux9ueCR569f745Fq9hQYhKLzjJbgt7RShZ2KEq3TPCgYNTtwpqjxE6SZAxyhhWx4nCiebTr3bxagJiV9AtcYM6FTJyyzokWgFGxtV9JHq","token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6ImY1ZDliNmVjLTc1ZTMtNGJmNi05OGYwLTFlZWViMTU1N2QyMiIsInR5cGUiOiJ1c2VyIiwiYXR0cmlidXRlcyI6eyJ1c2VybmFtZSI6InRlYXBvdCIsImVtYWlsIjoiaS5hbS5hQHRlYS5wb3QiLCJldGhBZGRyZXNzIjoiMHgyMTJjOTBmZGY5MGJiZDVlOWIzNTJiOWQyYjA4NmYyNjY2Y2ZlZWQ2Iiwibm9kZUFkZHJlc3MiOiJ4cHViNkR1eDl1ZUNSNTY5Zjc0NUZxOWhRWWhLTHpqSmJndDdSU2haMktFcTNUUENnWU5UdHdwcWp4RTZTWkF4eWhoV3g0bkNpZWJUcjNieGFnSmlWOUF0Y1lNNkZUSnl5em9rV2dGR3h0VjlKSHEifSwicmVsYXRpb25zaGlwcyI6e30sImlhdCI6MTU2Njg0OTc0NywiZXhwIjoxNTk4NDA3MzQ3fQ.IS79e9vBqRQUxberWaeXqOJgNkTspAIFNbJP0SNFOY0"}}}] [headers:{"host":"localhost:9001","connection":"keep-alive","content-length":"256","sec-fetch-mode":"cors","authorization":"Signature 0x41358b140924de8ed5d231b5fed958167bc358fe6945e9d129a5000db6219e677cdfc7a0f6735313e17ffe9c0dda7f654b81a72f82259f9bf690dc32132e73091c","user-agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/76.0.3809.100 Chrome/76.0.3809.100 Safari/537.36","content-type":"application/json; charset=utf-8","accept":"*/*","sec-fetch-site":"same-site","referer":"http://localhost:3334/setup/register","accept-encoding":"gzip, deflate, br","accept-language":"en-US,en;q=0.9"}]
@counterfactual/simple-hub-server: [timestamp:2019-08-26T20:02:27.504Z] [level:info] [message:OutgoingResponse] [status:201] [body:{"data":{"id":"f5d9b6ec-75e3-4bf6-98f0-1eeeb1557d22","type":"user","attributes":{"username":"teapot","email":"i.am.a@tea.pot","ethAddress":"0x212c90fdf90bbd5e9b352b9d2b086f2666cfeed6","nodeAddress":"xpub6Dux9ueCR569f745Fq9hQYhKLzjJbgt7RShZ2KEq3TPCgYNTtwpqjxE6SZAxyhhWx4nCiebTr3bxagJiV9AtcYM6FTJyyzokWgFGxtV9JHq","token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6ImY1ZDliNmVjLTc1ZTMtNGJmNi05OGYwLTFlZWViMTU1N2QyMiIsInR5cGUiOiJ1c2VyIiwiYXR0cmlidXRlcyI6eyJ1c2VybmFtZSI6InRlYXBvdCIsImVtYWlsIjoiaS5hbS5hQHRlYS5wb3QiLCJldGhBZGRyZXNzIjoiMHgyMTJjOTBmZGY5MGJiZDVlOWIzNTJiOWQyYjA4NmYyNjY2Y2ZlZWQ2Iiwibm9kZUFkZHJlc3MiOiJ4cHViNkR1eDl1ZUNSNTY5Zjc0NUZxOWhRWWhLTHpqSmJndDdSU2haMktFcTNUUENnWU5UdHdwcWp4RTZTWkF4eWhoV3g0bkNpZWJUcjNieGFnSmlWOUF0Y1lNNkZUSnl5em9rV2dGR3h0VjlKSHEifSwicmVsYXRpb25zaGlwcyI6e30sImlhdCI6MTU2Njg0OTc0NywiZXhwIjoxNTk4NDA3MzQ3fQ.IS79e9vBqRQUxberWaeXqOJgNkTspAIFNbJP0SNFOY0"},"relationships":{}}}] [headers:{"host":"localhost:9001","connection":"keep-alive","content-length":"256","sec-fetch-mode":"cors","authorization":"Signature 0x41358b140924de8ed5d231b5fed958167bc358fe6945e9d129a5000db6219e677cdfc7a0f6735313e17ffe9c0dda7f654b81a72f82259f9bf690dc32132e73091c","user-agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/76.0.3809.100 Chrome/76.0.3809.100 Safari/537.36","content-type":"application/json; charset=utf-8","accept":"*/*","sec-fetch-site":"same-site","referer":"http://localhost:3334/setup/register","accept-encoding":"gzip, deflate, br","accept-language":"en-US,en;q=
```

### Multisig deployment
``` 
@counterfactual/simple-hub-server: [timestamp:2019-08-26T20:02:36.777Z] [level:info] [message:IncomingRequest] [url:/api/multisig-deploys] [queryParams:undefined] [body:{"data":{"type":"multisigDeploy","attributes":{"ethAddress":"0x212c90fdf90bbd5e9b352b9d2b086f2666cfeed6"}}}] [headers:{"host":"localhost:9001","connection":"keep-alive","content-length":"107","sec-fetch-mode":"cors","user-agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/76.0.3809.100 Chrome/76.0.3809.100 Safari/537.36","content-type":"application/json; charset=utf-8","accept":"*/*","sec-fetch-site":"same-site","referer":"http://localhost:3334/setup/register","accept-encoding":"gzip, deflate, br","accept-language":"en-US,en;q=0.9"}]
@counterfactual/simple-hub-server: [timestamp:2019-08-26T20:02:36.777Z] [level:info] [message:OutgoingResponse] [status:201] [body:{"data":{"type":"multisig-deploy"}}]
```

**Comment by cf19drofxots** [comment]
@joelalejandro managed to repro this and put a fix for it (#2316)

**Comment by joelalejandro** [comment]
Closed by #2316.
